packages/expo-battery/android/src/main/java/expo/modules/battery/BatteryPackage.kt

```diff
@@ -0,0 +1,11 @@
+package expo.modules.battery
+
+import android.content.Context
+import org.unimodules.core.BasePackage
+import org.unimodules.core.ExportedModule
+
+class BatteryPackage : BasePackage() {
+  override fun createExportedModules(context: Context): List<ExportedModule> {
+    return listOf(BatteryModule(context) as ExportedModule)
```

Comment:
```suggestion
    return listOf(BatteryModule(context))
```
I belive this cast isn't needed 🤔